archivessnake/notes_csv.py

```diff
@@ -7,40 +7,52 @@
 import time
 
 from asnake.aspace import ASpace
+from asnake.utils import walk_tree
 
 config = ConfigParser()
 config.read("local_settings.cfg")
 
 aspace = ASpace(baseurl=config.get('ArchivesSpace', 'baseURL'), username=config.get('ArchivesSpace', 'user'), password=config.get('ArchivesSpace', 'password'))
+client = aspace.client
```

Comment:
You could streamline this by just doing:
```suggestion
client = ASpace(baseurl=config.get('ArchivesSpace', 'baseURL'), username=config.get('ArchivesSpace', 'user'), password=config.get('ArchivesSpace', 'password')).client
```